test/integration/scheduler/preemption/preemption_test.go

```diff
@@ -512,6 +522,14 @@ func TestAsyncPreemption(t *testing.T) {
 		// You should give a Pod index representing the order of Pod creation.
 		// e.g., if you want to check the Pod created first in the test case, you should give 0.
 		podRunningPreemption *int
+		// activatePod moves the pod from unschedulable to active or backoff.
+		activatePod *activatePod
```

Comment:
Could be just:
```suggestion
		activatePod string
```
Where you could pass a pod name